lib/shopify_cli/theme/syncer/standard_reporter.rb

```diff
@@ -24,7 +24,7 @@ def enable!
         end
 
         def report(message)
-          ctx.puts(message) if @enabled
+          ctx.error(message) if @enabled
```

Comment:
Should we also change the doc? Doc says standard output. Do we want `log` and `report` to be separate methods?

Reply:
Good catch! 🚀  I've updated the doc. Also, regarding the `report` method, reporters (we have more than one) are utility classes used by the `Syncer`, so it can control when status messages are reported and when they are ignored (with that in mind, I personally would keep the `report` method).